internal/shim-sgx/src/handler/mod.rs

```diff
@@ -255,6 +255,10 @@ impl<'a> Handler<'a> {
         buf: usize,
         buf_len: usize,
     ) -> Result<[usize; 2], c_int> {
+        if cfg!(feature = "disable-sgx-attestation") {
```

Comment:
nit; Why a macro and not a compile-time attribute (#[cfg(feature = "disable-sgx-attestation")])?
Is it because of the "unreachable code" warnings?

Reply:
yes